lua/pl/compat.lua

```diff
@@ -25,9 +27,20 @@ end
 function compat.execute (cmd)
     local res1,_,res3 = os.execute(cmd)
     if compat.lua51 and not compat.jit52 then
-        return res1==0,res1
+        if compat.isWindows then
+            res1 = res1 > 255 and res1 % 256 or res1
+            return res1==0,res1
+        else
+            res1 = res1 > 255 and res1 / 256 or res1
```

Comment:
@Tieske the LuaJIT 2.1 and LuaJIT 5.2 compatibility builds added to appveyor and travis (and they do seem to work as well)